examples/ndarray/animated_plot.py

```diff
@@ -54,7 +54,7 @@ def get_memory_mb():
 # --- 3D Data Generation ---
 def compute_3Ddata():
     time_factor = X * Y * 0.001
-    R = np.sqrt(Y**2 + Z**2)
+    R = np.hypot(Y, Z)
```

Comment:
That's a good point.  However, this does not work with blosc2 (yet) because it is not implemented in numexpr.  In addition, performance with numpy is basically the same.